### DISCUSSION TITLE
Encoder options for a quicker encoding

### DISCUSSION THREAD

**Discussion by remul19** [fraunhoferhhi/vvenc#286] [Q&A]
Hi All,

In my Linux machine I encoded a 4K test vector (using `--preset faster` option) but it took a very long time for the encoding to complete. Is there any other method to reduce the encoding time?

The encoder option which I used is :  `vvencapp --preset faster -i Input_3840x2160_50fps_10bit.yuv -s 3840x2160 -r 50 --internal-bitdepth 8 -o output.h266`

**Comment by remul19** [comment]
Hi ..  I removed the` --internal-bitdepth 8` option and encoding time reduced by nearly 40%. I am exploring for more options. Please let me know, if any other options can be tried out.

**Reply by adamjw24** [reply]
Your command is wrong for now. You need to add `-c yuv420_10` to your command line. Your input is 10 bit and the encoder app assumes 8 bit input (even if defaulting to 10 bit internal bit depth). Look at the decoded output, it should be wrong.
